app/pages/project/index.cjsx

```diff
@@ -125,7 +133,8 @@ ProjectPageController = React.createClass
             awaitProjectAvatar,
             awaitProjectCompleteness,
             awaitProjectRoles,
-            awaitPreferences
+            awaitPreferences,
+            this.props.actions.translations.load('project', project.id, this.props.translations.locale)
```

Comment:
Should `this.` be `@`

Reply:
I'm not too picky on adhering to coffeescript conventions or shorthand these days, as long as it doesn't break compiling, since eventually this should be rewritten in javascript anyway.